ui/admin/app/templates/scopes/scope/roles/role/scopes.hbs

```diff
@@ -21,137 +21,192 @@
-                  @model={{B.data.scope.id}}
-                  @icon='globe'
-                  @iconPosition='leading'
-                >
-                  {{#if B.data.scope.name}}
-                    {{t 'titles.global'}}
+                  {{#if (can 'read model' B.data)}}
+                    <LinkTo @route='scopes.scope' @model={{B.data}}>
+                      {{#if B.data.isGlobal}}
+                        {{t 'titles.global'}}
+                      {{else}}
+                        {{B.data.displayName}}
+                      {{/if}}
+                    </LinkTo>
                   {{else}}
-                    {{B.data.scope.id}}
+                    {{B.data.displayName}}
                   {{/if}}
-                </Hds::Link::Inline>
-              {{else if B.data.scope.isOrg}}
-                <Hds::Link::Inline
-                  @route='scopes.scope'
-                  @model={{B.data.scope.id}}
-                  @icon='org'
-                  @iconPosition='leading'
-                >
-                  {{#if B.data.scope.name}}
-                    {{B.data.scope.name}}
-                  {{else}}
-                    {{B.data.scope.id}}
-                  {{/if}}
-                </Hds::Link::Inline>
-              {{else}}
-                {{#if B.data.scope.name}}
-                  {{B.data.scope.name}}
-                {{else}}
-                  {{B.data.scope.id}}
                 {{/if}}
-              {{/if}}
-            </B.Td>
-            <B.Td>
-              {{#if B.data.type}}
-                <Hds::Badge
-                  @text={{t (concat 'resources.scope.types.' B.data.type)}}
+              </B.Td>
+              <B.Td>
+                <Hds::Copy::Snippet
+                  @textToCopy={{B.data.id}}
+                  @color='secondary'
                 />
-              {{/if}}
-            </B.Td>
-          </B.Tr>
-        </:body>
-      </Hds::Table>
+              </B.Td>
+              <B.Td>
+                {{#if B.data.scope.isGlobal}}
+                  <Hds::Link::Inline
+                    @route='scopes.scope'
+                    @model={{B.data.scope.id}}
+                    @icon='globe'
+                    @iconPosition='leading'
+                  >
+                    {{~if
+                      B.data.scope.name
+                      (t 'titles.global')
+                      B.data.scope.id
+                    }}
+                  </Hds::Link::Inline>
+                {{else if B.data.scope.isOrg}}
+                  <Hds::Link::Inline
+                    @route='scopes.scope'
+                    @model={{B.data.scope.id}}
+                    @icon='org'
+                    @iconPosition='leading'
+                  >
+                    {{~if B.data.scope.name B.data.scope.name B.data.scope.id}}
+                  </Hds::Link::Inline>
+                {{else}}
+                  {{if B.data.scope.name B.data.scope.name B.data.scope.id}}
```

Comment:
non-blocking: Why do we need the `~` on line 167 but not here?

Reply:
good point, tho now that i think about it i don't think this else is necessary at all since in every case the parent scope will either be org or global